[2025-10-16T10:21:11.348+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-16T10:21:11.378+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag2.dag2_python_1 scheduled__2025-10-16T10:20:00+00:00 [queued]>
[2025-10-16T10:21:11.388+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag2.dag2_python_1 scheduled__2025-10-16T10:20:00+00:00 [queued]>
[2025-10-16T10:21:11.390+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-16T10:21:11.465+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): dag2_python_1> on 2025-10-16 10:20:00+00:00
[2025-10-16T10:21:11.488+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag2', 'dag2_python_1', 'scheduled__2025-10-16T10:20:00+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpfl6ehoml']
[2025-10-16T10:21:11.492+0000] {standard_task_runner.py:91} INFO - Job 87: Subtask dag2_python_1
[2025-10-16T10:21:11.499+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=343) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-16T10:21:11.501+0000] {standard_task_runner.py:63} INFO - Started process 345 to run task
[2025-10-16T10:21:11.540+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-16T10:21:11.604+0000] {task_command.py:426} INFO - Running <TaskInstance: dag2.dag2_python_1 scheduled__2025-10-16T10:20:00+00:00 [running]> on host bb861ccd12fb
[2025-10-16T10:21:11.764+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag2' AIRFLOW_CTX_TASK_ID='dag2_python_1' AIRFLOW_CTX_EXECUTION_DATE='2025-10-16T10:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-16T10:20:00+00:00'
[2025-10-16T10:21:11.767+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-16T10:21:11.769+0000] {logging_mixin.py:188} INFO - bash you have 0 age
[2025-10-16T10:21:11.770+0000] {logging_mixin.py:188} INFO - bash you have 1 age
[2025-10-16T10:21:11.770+0000] {logging_mixin.py:188} INFO - bash you have 2 age
[2025-10-16T10:21:11.771+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-16T10:21:11.773+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-16T10:21:11.784+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag2, task_id=dag2_python_1, run_id=scheduled__2025-10-16T10:20:00+00:00, execution_date=20251016T102000, start_date=20251016T102111, end_date=20251016T102111
[2025-10-16T10:21:11.848+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-16T10:21:11.869+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-16T10:21:12.021+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
